File: SDKs/Aspose.Tasks-Cloud-SDK-for-Java/src/main/java/com/aspose/tasks/model/ResourceItems.java
package com.aspose.tasks.model;

import java.util.*;
import com.aspose.tasks.model.Link;
public class ResourceItems {
  private List<ResourceItem> List = new ArrayList<ResourceItem>();
  private Link link = null;
  /**
	 * getList
	 * Gets List<ResourceItem>
	 * @return List
	 */
  public List<ResourceItem> getList() {
    return List;
  }

	/**
	 * setList
	 * Sets List<ResourceItem>
	 * @param List List<ResourceItem>
	 */
  public void setList(List<ResourceItem> List) {
    this.List = List;
  }

  /**
	 * getLink
	 * Gets Link
	 * @return link
	 */
  public Link getLink() {
    return link;
  }

	/**
	 * setLink
	 * Sets Link
	 * @param link Link
	 */
  public void setLink(Link link) {
    this.link = link;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ResourceItems {\n");
    sb.append("  List: ").append(List).append("\n");
    sb.append("  link: ").append(link).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Tasks-Cloud-SDK-for-.NET/src/Com/Aspose/Tasks/Model/ResourceItems.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Tasks.Model {
  public class ResourceItems {
    public List<ResourceItem> List { get; set; }

    public Link link { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ResourceItems {\n");
      sb.Append("  List: ").Append(List).Append("\n");
      sb.Append("  link: ").Append(link).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
